### ISSUE TITLE
The Slew flag from hud_gauges.tbl causes 3D elements like the 3D radar or target model to distort when looking off-bore.

### ISSUE THREAD

**Issue by FrikgFeek** [scp-fs2open/fs2open.github.com#1584]
If Slew is set to YES for the radar(and the 3D radar flag is used) and/or the targetbox the elements with Slew set to YES will start distorting when reaching the edge of the screen before going to the top-left corner once the rest of the HUD disappears.

Here's an example 
https://www.youtube.com/watch?v=FHgH_jnLiFM

Note the radar "flattening" and the target model escaping the target box before both end up in the top left corner.

**Comment by JohnAFernandez** [comment]
I'm not that familiar with Slew. Is the radar supposed to be in the top left corner after the rest of the hud disappears?

**Comment by FrikgFeek** [comment]
No, it's definitely not supposed to do that. The Slew flag is supposed to make HUD elements act like they're projected onto a real cockpit so when you move the camera without moving the ship their position stays where they would be relative to the ship. Ie they move in the opposite direction of camera movement and eventually disappear off-screen.

The 3D radar and target box model going to the top left corner is not intended.
